impl/src/main/java/org/ehcache/internal/store/heap/OnHeapStore.java

```diff
@@ -367,6 +367,16 @@ public void clear() throws CacheAccessException {
 
   @Override
   public void invalidate() throws CacheAccessException {
```

Comment:
With the change of contract, the call from `CacheStore.clear()` seems wrong. We may need to add a `clear()` method on the `CachingTier` interface. Others?